doc/releases/migration-guide-4.2.rst

```diff
@@ -44,7 +44,8 @@ Boards
 * The DT binding :dtcompatible:`zephyr,native-posix-cpu` has been deprecated in favor of
   :dtcompatible:`zephyr,native-sim-cpu`.
 
-* Zephyr now supports version 1.11.1 of the :zephyr:board:`neorv32`.
+* Zephyr now supports version 1.11.1 of the :zephyr:board:`neorv32`. NEORV32 processor templates are
+  now supported via board variants.
```

Comment:
I might have missed the original bullet point but I don't see how it, nor the addition you're making to it, belons to the migration guide? You might have meant to highlight this in the release notes instead?
That being said, a migration guide entry indicating how to stop using the now deprecated ``neorv32`` board name/target would be useful.

Reply:
Thank you for reviewing.

I think they both belong in the migration guide. I have tried to clarify the text of the existing bullet, which should hopefully clarify why. Please have another look and provide feedback on the wording.